\hypertarget{classpip_1_1__internal_1_1exceptions_1_1InstallationError}{}\section{pip.\+\_\+internal.\+exceptions.\+Installation\+Error Class Reference}
\label{classpip_1_1__internal_1_1exceptions_1_1InstallationError}\index{pip.\+\_\+internal.\+exceptions.\+Installation\+Error@{pip.\+\_\+internal.\+exceptions.\+Installation\+Error}}


Inheritance diagram for pip.\+\_\+internal.\+exceptions.\+Installation\+Error\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=350pt]{classpip_1_1__internal_1_1exceptions_1_1InstallationError__inherit__graph}
\end{center}
\end{figure}


Collaboration diagram for pip.\+\_\+internal.\+exceptions.\+Installation\+Error\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=202pt]{classpip_1_1__internal_1_1exceptions_1_1InstallationError__coll__graph}
\end{center}
\end{figure}


\subsection{Detailed Description}
\begin{DoxyVerb}General exception during installation\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/lib/python3.\+9/site-\/packages/pip/\+\_\+internal/\hyperlink{pip_2__internal_2exceptions_8py}{exceptions.\+py}\end{DoxyCompactItemize}
